license/license.go

```diff
@@ -24,6 +24,7 @@ var EEAnalyzer = &analysis.Analyzer{
 func run(pass *analysis.Pass) (interface{}, error) {
 	const mockeryHeader = "// Code generated by mockery"
 	const goGenerateHeader = "//go:generate"
+	const bindataHeader = "(@generated)"
```

Comment:
What is going to be the bindataHeader that we are going to use? looks like `(@generated)` is a bit more generic than what I would like. I think is more explicit the `mockeryHeader` value, for example.

Reply:
@jespino , sure thing, bindata's gen header is a bit dynamic, since it outputs the package name. I can use a regex instead if you prefer, 0/5 here. i can argue that `//go:genrate` has the same vagueness as `(@generated)`.

The bindata header looks like this:

```
// Code generated for package {package-name} by go-bindata DO NOT EDIT. (@generated)
```

Reply:
Yes, I think i prefer that suffix, is less error prone.

The //go:generate is just saying that when "go:generate" is there, the license can be positioned in other place, so make it "generic" is right, in your case is a bit more error prone IMO.